TriblerGUI/widgets/trustpage/js/parser.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Parse a JSON string into the defined format. This file explains what JSON is expected and how the resulting
+ * javascript objects are constructed. The type definitions can be used for type hinting.
+ *
+ * The expected JSON format:
+ *
+ * {
+ *       "focus_node": "xyz",
+ *       "neighbor_level": 1,
+ *       "nodes": [{
+ *           "public_key": "xyz",
+ *           "total_up": 12736457,
+ *           "total_down": 1827364,
+ *           "page_rank": 0.5
+ *       }, ...],
+ *       "edges": [{
+ *           "from": "xyz",
+ *           "to": "xyz_n1",
+ *           "amount": 12384
+ *       }, ...]
+ * }
+ *
+ * This will be converted into the following object:
+ */
+
+/**
+ * Convert a JSON string into a GraphResponseData Object
+ * @param {String} json - The JSON string assumed to have the defined format
+ * @returns {GraphResponseData} - The response object
+ */
+function parse(json){
+    return JSON.parse(json);
+}
```

Comment:
There are only 1 line of code in this whole file, don't you think we are able to place this file in another file or add something more to it?